components/event_dispatcher.rst

```diff
@@ -135,7 +135,7 @@ A call to the dispatcher's ``addListener()`` method associates any valid
 PHP callable to an event::
 
     $listener = new AcmeListener();
-    $dispatcher->addListener('acme.action', array($listener, 'onFooAction'));
+    $dispatcher->addListener('foo.action', array($listener, 'onFooAction'));
```

Comment:
I guess here `foo` is a placeholder for `acme` and `action` a placeholder for a verb, so maybe `acme.foo` would be less confusing?